test/jdk/java/awt/Focus/UnaccessibleChoice/AccessibleChoiceTest.java

```diff
@@ -113,25 +120,30 @@ public void start () throws IOException {
             throw new RuntimeException("Interrupted !!!");
         }
 
-        if (!def.isFocusOwner()) {
+        if (!button.isFocusOwner()) {
             throw new RuntimeException("Button doesn't have focus");
         }
 
+        robot.delay(500);
+
         // Press Tab key to move focus to Choice
         robot.keyPress(KeyEvent.VK_TAB);
         robot.keyRelease(KeyEvent.VK_TAB);
 
         robot.delay(500);
 
-        // Press Down key to select next item in the choice(Motif 2.1)
+        if (!choice.isFocusOwner()) {
+            throw new RuntimeException("Choice doesn't have focus");
+        }
+
+        // Press Down key to select next item in the choice
         // If bug exists we won't be able to do so
         robot.keyPress(KeyEvent.VK_DOWN);
         robot.keyRelease(KeyEvent.VK_DOWN);
 
-        robot.delay(500);
-
         String osName = System.getProperty("os.name").toLowerCase();
         if (osName.startsWith("mac")) {
```

Comment:
Suggestion: Platform.isOSX() looks cleaner. @library tag is required to use jdk.test.lib.Platform